geoclustering/coclustering_numpy_lowmem.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
```

Comment:
We can remove this file?

Reply:
yes, this can go. It's legacy from before number